nautobot/core/views/renderers.py

```diff
@@ -158,7 +158,10 @@ def get_context(self, data, accepted_media_type, renderer_context):
             if view.action == "list":
                 if view.filterset_class is not None:
                     view.queryset = view.filter_queryset(view.get_queryset())
-                    filterset_filters = view.filterset.get_filters()
+                    if view.filterset is not None:
+                        filterset_filters = view.filterset.filters
+                    else:
+                        filterset_filters = view.filterset.get_filters()
```

Comment:
This appears as if the `else` will never be called. So it seems like `view.filter_queryset()` has a side effect of setting `view.filterset`... 😬

Reply:
Side-effect from line 60 but it makes it's own assumptions that `view.filterset` was set... 🤷